aya/src/programs/cgroup_skb.rs

```diff
@@ -1,7 +1,7 @@
 //! Cgroup skb programs.
 use std::{
     hash::Hash,
-    os::unix::prelude::{AsRawFd, RawFd},
+    os::fd::{AsRawFd, RawFd},
```

Comment:
Unless strictly necessary, it would be better to make these changes in another PR as it's not a nice diff.

Reply:
https://github.com/aya-rs/aya/pull/633

Reply:
Done.